lib/line/bot/client.rb

```diff
@@ -547,6 +547,18 @@ def create_rich_menu(rich_menu)
         post(endpoint, endpoint_path, rich_menu.to_json, credentials)
       end
 
+      # Validate a rich menu
```

Comment:
[nice to have]

This endpoint validates an object of the rich menu.

```suggestion
      # Validate a rich menu object
```